packages/core/src/services/essentials/dom-utils.ts

```diff
@@ -393,140 +362,144 @@ export class DOMUtils extends Service {
 		) {
 			// Call the correct function depending on the browser
 			if (document.exitFullscreen) {
-				document.exitFullscreen();
+				document.exitFullscreen()
 			} else if (document['webkitExitFullscreen']) {
-				document['webkitExitFullscreen']();
+				document['webkitExitFullscreen']()
 			} else if (document['mozCancelFullScreen']) {
-				document['mozCancelFullScreen']();
+				document['mozCancelFullScreen']()
 			} else if (document['msExitFullscreen']) {
-				document['msExitFullscreen']();
+				document['msExitFullscreen']()
 			}
 		} else {
 			// Call the correct function depending on the browser
 			if (holder.requestFullscreen) {
-				holder.requestFullscreen();
+				holder.requestFullscreen()
 			} else if (holder.webkitRequestFullscreen) {
-				holder.webkitRequestFullscreen();
+				holder.webkitRequestFullscreen()
 			} else if (holder.mozRequestFullScreen) {
-				holder.mozRequestFullScreen();
+				holder.mozRequestFullScreen()
 			} else if (holder.msRequestFullscreen) {
-				holder.msRequestFullscreen();
+				holder.msRequestFullscreen()
 			}
 		}
 	}
 
 	handleFullscreenChange() {
 		document.addEventListener('fullscreenchange', () => {
-			const holderSelection = select(this.getHolder());
-			const isFullScreen = holderSelection.classed('fullscreen');
+			const holderSelection = select(this.getHolder())
+			const isFullScreen = holderSelection.classed('fullscreen')
 
 			// Toggle the `fullscreen` classname
-			holderSelection.classed('fullscreen', !isFullScreen);
-		});
+			holderSelection.classed('fullscreen', !isFullScreen)
+		})
 	}
 
 	verifyCSSStylesBeingApplied() {
+		// This code was triggering erroneous messages even when styles were loaded
 		// setTimeout is needed here since in `addSVGElement()` we're appending the
 		// CSS verifier element, and need to allow some time for it to become available
 		// in the DOM
 		setTimeout(() => {
-			const cssVerifierElement = select(this.mainContainer)
-				.select(`g.${CSS_VERIFIER_ELEMENT_CLASSNAME}`)
-				.node();
-			const computedStyles = getComputedStyle(cssVerifierElement as any);
-			if (
+			/*const cssVerifierElement = */ select(this.mainContainer)
+				.select(`g.${CSS_VERIFIER_ELEMENT_CLASSNAME}`) // BUG: weird to add a <g> element to a <div> (perhaps change it to a comment?)
+				.node()
+			/*
+			const computedStyles = getComputedStyle(cssVerifierElement as any)
+
+			if ( // true even when styles are properly loaded
 				computedStyles.getPropertyValue('overflow') !== 'hidden' ||
 				computedStyles.getPropertyValue('opacity') !== '0'
 			) {
 				console.error(
 					'Missing CSS styles for Carbon Charts. Please read the Carbon Charts getting started guide.'
-				);
+				)
```

Comment:
Are we removing this piece?

Reply:
It's logging messages that there are missing CSS styles even when they are loaded as documented. Also, it's creating a `g` tag within the `div` which is not really a valid child for anything other than an `svg`.

A few options...

1. Fix the test to see if the required CSS is properly loaded (perhaps check for the presence of CSS custom properties) instead of overflow and opacity.
2. Add a CSS verifier element but do it as a `span` or `div` instead of as a `g`
3. Do #1 and #2
4. Take out that verification altogether and trust the implementer to follow the documentation regarding loading of the styles.

Any preference?

Reply:
let's not take it out for now, that could be considered a breaking change. Can we just change it to a `span`?

I haven't seen it throw errors incorrectly ever yet... every time someone reported a false error, I look into it and realize that it is in fact correctly working. Did you find examples where it's actually not working as expected?